device/alif-ensemble/include/GpioSignal.hpp

```diff
@@ -24,7 +24,7 @@
 extern "C" {
 #endif // #if defined(__cplusplus)
 
-#include "gpio.h"
+#include "../src/gpio.h"
```

Comment:
This looks pretty unusual. Header files placed into the src folder are typically considered "private" while ones in include folder represent the "public interface".

Reply:
One of the type definitions required in this file was not present in the pack's `gpio.h` and only in `src/gpio.h`. Would it better to remove the include and simply move the type definition into this file itself?